hoomd/md/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ set(_md_headers ActiveForceComputeGPU.h
                 EvaluatorPairFourier.h
                 EvaluatorPairReactionField.h
                 EvaluatorPairExpandedLJ.h
+		EvaluatorPairTable.h
```

Comment:
Let's indent this line the same as all the other lines are indented.

Reply:
I realized the GitHub online editor defaults to tab indentation